packages/graphics/opengl-meson/package.mk

```diff
@@ -29,8 +29,12 @@ case $MESON_FAMILY in
     PKG_SHA256="21a8376668c84bf1b9e64a917fcfa1cf74689035fed8e4630833c9cde28d40c1"
     ;;
   gxbb)
-    PKG_VERSION="gxbb-r5p1-01rel0"
-    ;;
+    if  [ "$TARGET_ARCH" = "arm"  ];then
```

Comment:
Extra space after `"arm"` can be removed.

Reply:
Thanks. I will fix that.